integration/build/build_test.go

```diff
@@ -130,6 +130,17 @@ func TestBuildCommandV1(t *testing.T) {
 	}
 	require.NoError(t, commands.RunOktetoBuild(oktetoPath, options))
 	require.True(t, isImageBuilt(expectedImage))
+
+	options = &commands.BuildOptions{
```

Comment:
Can you move this to a new test function. We are parallelising by tests functions so if we add it in another one we will run the test in parallel